### PR TITLE
feat: Added Version Name in Device Settings

### LINKED ISSUE #1215: Show app version in Device Settings
### Feature Request

**Feature Description**

Show app version name in Device Settings.
   
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1215 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 

Screenshots for the change:
versn

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@sridharjajoo This is done in order to keep UI similar to EventBrite.

**Comment by Masquerade0097** [comment]
@iamareebjamal @sridharjajoo Please review.

File: app/src/main/res/xml/preferences.xml [hunk 1]
```diff
@@ -27,4 +27,10 @@
             android:summary="Love us? Rate us on Google Play." />
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/eventyay_organizer">
+        <Preference
+            android:selectable="false"
+            android:key="app_version" />
```
Comment (mishuvs) [inline app/src/main/res/xml/preferences.xml]: Same here

File: app/src/main/java/org/fossasia/openevent/app/core/settings/SettingsFragment.java [hunk 2]
```diff
@@ -37,6 +37,8 @@ public void onCreatePreferencesFix(@Nullable Bundle bundle, String rootKey) {
 
         setPreferencesFromResource(R.xml.preferences, rootKey);
 
+        findPreference("app_version").setTitle("Version " + BuildConfig.VERSION_NAME);
```
Comment (mishuvs) [inline app/src/main/java/org/fossasia/openevent/app/core/settings/SettingsFragment.java]: Use string resource for keys.

**Comment by Masquerade0097** [comment]
@mishuvs @iamareebjamal Please review.

**Pull request merged**